caillef/dustzh.lua

```diff
@@ -1631,3 +1705,42 @@ killfeedMetatable = {
 	}
 }
 setmetatable(killfeed, killfeedMetatable)
+
+-- display ammo on the left of the screen
+local ammoIndicators = {}
+local maxAmmoPerRow = 12
+local rowSpacing = 22
+
+addAmmoIndication = function(numberAmmo)
```

Comment:
maybe not urgent, but the ammo indicators (bullets) don't scale with the text either.